FinniversKit/Sources/DNA/Spacing/Spacing.swift

```diff
@@ -6,51 +6,58 @@ import Foundation
 
 public extension CGFloat {
     /// Separation of 2 points.
-    @available(*, deprecated, message: "Use spacingXXS instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing25 instead.")
     static let verySmallSpacing: CGFloat = 2
 
     /// Separation of 4 points.
-    @available(*, deprecated, message: "Use spacingXS instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing50 instead.")
     static let smallSpacing: CGFloat = 4
 
     /// Separation of 8 points.
-    @available(*, deprecated, message: "Use spacingS instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing100 instead.")
     static let mediumSpacing: CGFloat = 8
 
     /// Separation of 16 points.
-    @available(*, deprecated, message: "Use spacingM instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing200 instead.")
     static let mediumLargeSpacing: CGFloat = 16
 
     /// Separation of 24 points.
-    @available(*, deprecated, message: "Use spacingL instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing300 instead.")
     static let mediumPlusSpacing: CGFloat = 24
 
     /// Separation of 32 points.
-    @available(*, deprecated, message: "Use spacingXL instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing400 instead.")
     static let largeSpacing: CGFloat = 32
 
     /// Separation of 64 points.
-    @available(*, deprecated, message: "Use spacingXXL instead.")
+    @available(*, deprecated, message: "Use Warp.Spacing.spacing800 instead.")
     static let veryLargeSpacing: CGFloat = 64
```

Comment:
None of these already deprecated properties are used, so we can just remove them 🙂 I had no idea we still had them in the repo😅